templates/zabbix-agent2-debian.init.erb

```diff
@@ -0,0 +1,68 @@
+#! /bin/sh
```

Comment:
are there any supported distributions that still use this legacy style and not the systemd unit?

Reply:
Probably not! Should I remove these files and relevant code?

Reply:
yes please. let's keep the complexity to the minimal required one.

Reply:
I've cleaned up the files and set the manage_startup_script variable to default to false in params.pp. However, I see some of the tests are failing now because systemd is the only supported startup type. I've looked at fixing the tests for this, but I fear I'm way over my head. 🙄